packages/lesswrong/lib/collections/posts/callbacks.js

```diff
@@ -41,6 +48,22 @@ function PostsSetPostedAt (modifier, post) {
 }
 addCallback("posts.undraft.sync", PostsSetPostedAt);
 
+/**
+ * @summary increment postCount when post is undrafted
+ */
+function postsUndraftIncrementPostCount (post, oldPost) {
+  Users.update({_id: post.userId}, {$inc: {postCount: 1}})
+}
+addCallback("posts.undraft.async", postsUndraftIncrementPostCount);
+
+/**
+ * @summary decrement postCount when post is drafted
+ */
+function postsDraftDecrementPostCount (post, oldPost) {
+  Users.update({_id: post.userId}, {$inc: {postCount: -1}})
+}
+addCallback("posts.draft.async", postsDraftDecrementPostCount);
+
 /**
  * @summary update frontpagePostCount when post is moved into frontpage
```

Comment:
@Raemon While working on this I noticed that a similar issue exists for `frontpagePostCount` when converting frontpage posts back to drafts. Should I open another issue to fix that?

Reply:
If you've noticed it all at once at the issues are tightly coupled, I think it makes sense to just go ahead and fix them all at once. (I'm ambivalent about it though)

Reply:
Let's do it in a separate issue. It doesn't look like frontpagePostCount is being used, but if you ever do start using it we can always fix it then.